subsys/bluetooth/mesh/shell.c

```diff
@@ -3335,227 +1215,146 @@ static int cmd_cdb_app_key_del(const struct shell *shell, size_t argc,
 
 	app_idx = shell_strtoul(argv[1], 0, &err);
 	if (err) {
-		shell_warn(shell, "Unable to parse input string argument");
+		shell_warn(sh, "Unable to parse input string argument");
 		return err;
 	}
 
 	key = bt_mesh_cdb_app_key_get(app_idx);
 	if (key == NULL) {
-		shell_print(shell, "No AppKey 0x%03x", app_idx);
+		shell_print(sh, "No AppKey 0x%03x", app_idx);
 		return 0;
 	}
 
 	bt_mesh_cdb_app_key_del(key, true);
 
-	shell_print(shell, "Deleted AppKey 0x%03x", app_idx);
+	shell_print(sh, "Deleted AppKey 0x%03x", app_idx);
 
 	return 0;
 }
 #endif
 
-/* List of Mesh subcommands.
- *
- * Each command is documented in doc/reference/bluetooth/mesh/shell.rst.
- *
- * Please keep the documentation up to date by adding any new commands to the
- * list.
- */
-SHELL_STATIC_SUBCMD_SET_CREATE(mesh_cmds,
-	/* General operations */
-	SHELL_CMD_ARG(init, NULL, NULL, cmd_init, 1, 0),
-	SHELL_CMD_ARG(reset, NULL, "<addr>", cmd_reset, 2, 0),
-#if defined(CONFIG_BT_MESH_LOW_POWER)
-	SHELL_CMD_ARG(lpn, NULL, "<value: off, on>", cmd_lpn, 2, 0),
-	SHELL_CMD_ARG(poll, NULL, NULL, cmd_poll, 1, 0),
-#endif
-#if defined(CONFIG_BT_MESH_GATT_PROXY)
-	SHELL_CMD_ARG(ident, NULL, NULL, cmd_ident, 1, 0),
-#endif
-	SHELL_CMD_ARG(dst, NULL, "[destination address]", cmd_dst, 1, 1),
-	SHELL_CMD_ARG(netidx, NULL, "[NetIdx]", cmd_netidx, 1, 1),
-	SHELL_CMD_ARG(appidx, NULL, "[AppIdx]", cmd_appidx, 1, 1),
-
-	/* Commands which access internal APIs, for testing only */
-	SHELL_CMD_ARG(net-send, NULL, "<hex string>", cmd_net_send, 2, 0),
-#if defined(CONFIG_BT_MESH_IV_UPDATE_TEST)
-	SHELL_CMD_ARG(iv-update, NULL, NULL, cmd_iv_update, 1, 0),
-	SHELL_CMD_ARG(iv-update-test, NULL, "<value: off, on>",
-		      cmd_iv_update_test, 2, 0),
+#if defined(CONFIG_BT_MESH_CDB)
+SHELL_STATIC_SUBCMD_SET_CREATE(cdb_cmds,
+	/* Mesh Configuration Database Operations */
+	SHELL_CMD_ARG(create, NULL, "[NetKey]", cmd_cdb_create, 1, 1),
+	SHELL_CMD_ARG(clear, NULL, NULL, cmd_cdb_clear, 1, 0),
+	SHELL_CMD_ARG(show, NULL, NULL, cmd_cdb_show, 1, 0),
+	SHELL_CMD_ARG(node_add, NULL, "<UUID> <Addr> <Num-elem> "
+		      "<NetKeyIdx> [DevKey]", cmd_cdb_node_add, 5, 1),
+	SHELL_CMD_ARG(node_del, NULL, "<Addr>", cmd_cdb_node_del, 2, 0),
+	SHELL_CMD_ARG(subnet_add, NULL, "<NeyKeyIdx> [<NetKey>]",
+		      cmd_cdb_subnet_add, 2, 1),
+	SHELL_CMD_ARG(subnet_del, NULL, "<NetKeyIdx>", cmd_cdb_subnet_del,
+		      2, 0),
+	SHELL_CMD_ARG(app_key_add, NULL, "<NetKeyIdx> <AppKeyIdx> "
+		      "[<AppKey>]", cmd_cdb_app_key_add, 3, 1),
+	SHELL_CMD_ARG(app_key_del, NULL, "<AppKeyIdx>", cmd_cdb_app_key_del,
+		      2, 0),
+	SHELL_SUBCMD_SET_END);
 #endif
-	SHELL_CMD_ARG(rpl-clear, NULL, NULL, cmd_rpl_clear, 1, 0),
 
-	/* Provisioning operations */
+SHELL_STATIC_SUBCMD_SET_CREATE(prov_cmds,
+	SHELL_CMD_ARG(uuid, NULL, "<UUID: 1-16 hex values>", cmd_uuid, 2, 0),
+	SHELL_CMD_ARG(input_num, NULL, "<Number>", cmd_input_num, 2, 0),
```

Comment:
Im not sure if I understand. You meen that it could be: `SHELL_CMD_ARG(input num, NULL, "<Number>", cmd_input_num, 2, 0),`? Tried it for a command and it did not seem to work.

Reply:
I mean the previous syntax "input-num" will work here. `SHELL_SUBCMD_ADD` doesn't allow to use dash, but `SHELL_CMD_ARG` does allow.

Reply:
Thought it was the other way around 😅  Would you prefer dash?

Reply:
Dash (or hyphen) is close to traditional style for long command names.